src/xcode/ENA/ENA/Source/Scenes/AppInformation/AppInformationViewController.swift

```diff
@@ -12,7 +12,8 @@ class AppInformationViewController: DynamicTableViewController, NavigationBarOpa
 	init(
 		elsService: ErrorLogSubmissionProviding,
 		finishedDeltaOnboardings: [String: [String]],
-		cclService: CCLServable
+		cclService: CCLServable,
+		isHibernationState: Bool? = CWAHibernationProvider.shared.isHibernationState
```

Comment:
Why optional `Bool`?

Reply:
So we don't need to change the code at other places.